package.json

```diff
@@ -39,10 +39,11 @@
     "@sentry/nextjs": "^7.11.1",
     "@solana/buffer-layout": "^4.0.0",
     "@solana/qr-code-styling": "^1.6.0-beta.0",
-    "@solana/wallet-adapter-react": "^0.15.3",
-    "@solana/wallet-adapter-react-ui": "^0.9.5",
-    "@solana/wallet-adapter-wallets": "^0.15.1",
-    "@solana/web3.js": "^1.32.1",
+    "@solana/wallet-adapter-base": "^0.9.18",
+    "@solana/wallet-adapter-react": "rc",
+    "@solana/wallet-adapter-react-ui": "rc",
+    "@solana/wallet-adapter-wallets": "^0.19.5",
+    "@solana/web3.js": "^1.58.0",
```

Comment:
These are all the latest, except for Web3, which uses the minimum required by Wallet Adapter to support VersionedTransaction and related types.